<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="JATIN-PC" name="TestCases.Test_Random" tests="3" failures="1" timestamp="24 Nov 2014 08:27:12 GMT" time="64.345" errors="1">
  <testcase name="TestExistingToNew_LoginRandom" time="28.693" classname="TestCases.Test_Random">
    <error type="java.lang.AssertionError" message="expected [Shipping] but found [TOTAL]">
      <![CDATA[java.lang.AssertionError: expected [Shipping] but found [TOTAL]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:494)
	at org.testng.Assert.assertEquals(Assert.java:123)
	at org.testng.Assert.assertEquals(Assert.java:176)
	at org.testng.Assert.assertEquals(Assert.java:186)
	at core.GiftExistingToNew_Login.giftExistingUserToNewUser_Login_Random(GiftExistingToNew_Login.java:157)
	at TestCases.Test_Random.TestExistingToNew_LoginRandom(Test_Random.java:34)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
]]>
    </error>
  </testcase> <!-- TestExistingToNew_LoginRandom -->
  <testcase name="TestNewToExistingRandom" time="32.591" classname="TestCases.Test_Random"/>
  <testcase name="TestExistingToExisting_LoginRandom" time="3.061" classname="TestCases.Test_Random">
    <failure type="org.openqa.selenium.ElementNotVisibleException" message="Element is not currently visible and so may not be interacted with
Command duration or timeout: 20 milliseconds
Build info: version: &#039;2.42.2&#039;, revision: &#039;6a6995d&#039;, time: &#039;2014-06-03 17:42:30&#039;
System info: host: &#039;JATIN-PC&#039;, ip: &#039;192.168.107.1&#039;, os.name: &#039;Windows 7&#039;, os.arch: &#039;x86&#039;, os.version: &#039;6.1&#039;, java.version: &#039;1.7.0_65&#039;
Session ID: d3a05de9-49d8-458e-9135-938d85077ff4
Driver info: org.openqa.selenium.firefox.FirefoxDriver
Capabilities [{platform=XP, acceptSslCerts=true, javascriptEnabled=true, cssSelectorsEnabled=true, databaseEnabled=true, browserName=firefox, handlesAlerts=true, browserConnectionEnabled=true, webStorageEnabled=true, nativeEvents=false, rotatable=false, locationContextEnabled=true, applicationCacheEnabled=true, takesScreenshot=true, version=25.0.1}]">
      <![CDATA[org.openqa.selenium.ElementNotVisibleException: Element is not currently visible and so may not be interacted with
Command duration or timeout: 20 milliseconds
Build info: version: '2.42.2', revision: '6a6995d', time: '2014-06-03 17:42:30'
System info: host: 'JATIN-PC', ip: '192.168.107.1', os.name: 'Windows 7', os.arch: 'x86', os.version: '6.1', java.version: '1.7.0_65'
Session ID: d3a05de9-49d8-458e-9135-938d85077ff4
Driver info: org.openqa.selenium.firefox.FirefoxDriver
Capabilities [{platform=XP, acceptSslCerts=true, javascriptEnabled=true, cssSelectorsEnabled=true, databaseEnabled=true, browserName=firefox, handlesAlerts=true, browserConnectionEnabled=true, webStorageEnabled=true, nativeEvents=false, rotatable=false, locationContextEnabled=true, applicationCacheEnabled=true, takesScreenshot=true, version=25.0.1}]
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(Unknown Source)
	at java.lang.reflect.Constructor.newInstance(Unknown Source)
	at org.openqa.selenium.remote.ErrorHandler.createThrowable(ErrorHandler.java:204)
	at org.openqa.selenium.remote.ErrorHandler.throwIfResponseFailed(ErrorHandler.java:156)
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:599)
	at org.openqa.selenium.remote.RemoteWebElement.execute(RemoteWebElement.java:268)
	at org.openqa.selenium.remote.RemoteWebElement.click(RemoteWebElement.java:79)
	at core.Login.do_login(Login.java:24)
	at TestCases.Test_Random.TestExistingToExisting_LoginRandom(Test_Random.java:49)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
Caused by: org.openqa.selenium.remote.ErrorHandler$UnknownServerException: Element is not currently visible and so may not be interacted with
Build info: version: '2.42.2', revision: '6a6995d', time: '2014-06-03 17:42:30'
System info: host: 'JATIN-PC', ip: '192.168.107.1', os.name: 'Windows 7', os.arch: 'x86', os.version: '6.1', java.version: '1.7.0_65'
Driver info: driver.version: unknown
	at <anonymous class>.fxdriver.preconditions.visible(file:///C:/Users/JATIN/AppData/Local/Temp/anonymous6085939954897502271webdriver-profile/extensions/fxdriver@googlecode.com/components/command_processor.js:8791)
	at <anonymous class>.DelayedCommand.prototype.checkPreconditions_(file:///C:/Users/JATIN/AppData/Local/Temp/anonymous6085939954897502271webdriver-profile/extensions/fxdriver@googlecode.com/components/command_processor.js:11438)
	at <anonymous class>.DelayedCommand.prototype.executeInternal_/h(file:///C:/Users/JATIN/AppData/Local/Temp/anonymous6085939954897502271webdriver-profile/extensions/fxdriver@googlecode.com/components/command_processor.js:11455)
	at <anonymous class>.DelayedCommand.prototype.executeInternal_(file:///C:/Users/JATIN/AppData/Local/Temp/anonymous6085939954897502271webdriver-profile/extensions/fxdriver@googlecode.com/components/command_processor.js:11460)
	at <anonymous class>.DelayedCommand.prototype.execute/<(file:///C:/Users/JATIN/AppData/Local/Temp/anonymous6085939954897502271webdriver-profile/extensions/fxdriver@googlecode.com/components/command_processor.js:11402)
]]>
    </failure>
  </testcase> <!-- TestExistingToExisting_LoginRandom -->
</testsuite> <!-- TestCases.Test_Random -->
